overlord/snapstate/snapstate.go

```diff
@@ -1864,23 +1864,28 @@ type RemoveFlags struct {
 // Remove returns a set of tasks for removing snap.
 // Note that the state must be locked by the caller.
 func Remove(st *state.State, name string, revision snap.Revision, flags *RemoveFlags) (*state.TaskSet, error) {
+	ts, _, err := remove(st, name, revision, flags)
+	return ts, err
+}
+
+func remove(st *state.State, name string, revision snap.Revision, flags *RemoveFlags) (*state.TaskSet, uint64, error) {
```

Comment:
Hm, I think the return arguments now need to be named and I would also rename the function. Not sure how though :( Maybe getRemoveTasksSetAndSnapshotSize() ? But probably too long, let's see if others have good suggestions.

Reply:
I would just call it "removeTasks" but indeed add name to the return arguments and give a doc comment

// removeTasks provides the task set to remove snap name after taking a snapshot if flags.Purge is not true, it also computes an estimate of the latter size